docs/source/examples/ddd_on_ibmq_ghz.md

```diff
@@ -198,14 +192,20 @@ def ibm_executor(
 :tags: ["remove-output"]
 data = []
 for depth in depths:
-    for rule in rules:
-        print(f"DDD sequence: {rule}.")
-        circuit = get_circuit_with_sequence(depth, rule)
-        noisy_value = ibm_executor(
+    circuit = get_circuit(depth)
+    noisy_value = ibm_executor(
             circuit, shots=shots, correct_bitstring=[0]
+    )
+    print("Result:", noisy_value)
+    data.append((depth, rule, noisy_value))
```

Comment:
Does this run even if `rule` is not yet defined?

Reply:
I still don't know how it was able to run before without defining the `rule`, but it should be fixed now. 

Also changed the wording of the text in the beginning and now it reads, "the frequency of the $|0 \rangle$ bitstring is our target metric (in this example we only measure the first qubit)."
Not ideal but consistent at least 😅